### ISSUE TITLE
Single link for multi-citation

### ISSUE THREAD

**Issue by andriusvelykis** [inukshuk/jekyll-scholar#45]
When using multi-citation e.g. `{% cite ruby microscope %}`, the output HTML has only a single link to the first item in citations, as indicated in the README:

``` html
<a href="#ruby">(Flanagan &amp; Matsumoto 2008; Shaughnessy 2013)</a>
```

It would be nice to have support for multiple links - one for each citation. As discussed further in #41, the citations are rendered by cite processor and there needs to be a way to hook/hack into it.

Regarding "how it should look", I think one solution would be to always hook into the citeproc and add links for the cite item, but not parentheses, e.g. (<a href="#ruby">Flanagan &amp; Matsumoto 2008</a>; <a href="#microscope">Shaughnessy 2013</a>) or [<a href="#ruby">1</a>, <a href="#microscope">2</a>].

``` html
(<a href="#ruby">Flanagan &amp; Matsumoto 2008</a>; <a href="#microscope">Shaughnessy 2013</a>)
```

For single-cite items it would also change, becoming (<a href="#ruby">Flanagan &amp; Matsumoto 2008</a>) or [<a href="#ruby">1</a>].

``` html
(<a href="#ruby">Flanagan &amp; Matsumoto 2008</a>)
```

**Comment by reitzig** [comment]
Any progress (on the horizon) for this?

**Comment by inukshuk** [comment]
No, citeproc has no support for arbitrary hooks yet and no one's currently working on this as far as I know.

**Comment by leotrs** [comment]
Almost four years after the last comment, is there any chance of revisiting this any time soon?
